test/e2e/egressqos.go

```diff
@@ -229,6 +229,64 @@ spec:
 		},
 		ginkgotable.Entry("ipv4 pod", tcpdumpIPv4, &dstPod1IPv4, "/32", &dstPod2IPv4, "/32"),
 		ginkgotable.Entry("ipv6 pod", tcpdumpIPv6, &dstPod1IPv6, "/128", &dstPod2IPv6, "/128"))
+
+	ginkgo.It("Should deny resources with bad values", func() {
+		egressQoSConfig := fmt.Sprintf(`
+apiVersion: k8s.ovn.org/v1
+kind: EgressQoS
+metadata:
+  name: not-default
+  namespace: ` + f.Namespace.Name + `
+spec:
+  egress:
+  - dscp: 50
+`)
+
+		if err := ioutil.WriteFile(egressQoSYaml, []byte(egressQoSConfig), 0644); err != nil {
+			framework.Failf("Unable to write CRD config to disk: %v", err)
+		}
+
+		defer func() {
+			if err := os.Remove(egressQoSYaml); err != nil {
+				framework.Logf("Unable to remove the CRD config from disk: %v", err)
+			}
+		}()
+
+		framework.Logf("Create an EgressQoS not named default")
```

Comment:
change to:
~~~
ginkgo.By("Creating an EgressQoS not named default")
~~~
and move above `egressQoSConfig := fmt.Sprintf(``

Reply:
done